src/ui/CustomStyleEditor.js

```diff
@@ -887,7 +891,7 @@ class CustomStyleEditor extends React.PureComponent<any, any> {
                       disabled={this.state.styles.boldPartial ? false : true}
                       name="boldscentence"
                       onChange={this.onScentenceRadioChanged.bind(this)}
-                      style={{ marginLeft: '20px' }}
+                      style={{ marginLeft: '21px' }}
```

Comment:
Is 21 a font size? This looks like it should be in em instead of px.

Reply:
This changes the position of the radio button. I have changed the value, so that it looks organized.